tensorflow/tensorboard/dist/tf-tensorboard.html

```diff
@@ -3856,9 +3856,9 @@ <h3 id="tooltip-help" class="tooltip-container">
             var data = dataset.data();
             var kernelRadius = Math.floor(data.length * factor / 2);
             data.forEach(function (d, i) {
-                var actualKernelRadius = Math.min(kernelRadius, i, data.length - i - 1);
```

Comment:
There's no need to update tf-tensorboard by hand, it will get automatically regenerated in time for the next release.
We're working on removing this layer of indirection by directly compiling TensorBoard in bazel... it should come Soon(tm)